spec/helpers.rb

```diff
@@ -48,6 +48,10 @@ def get_text(doc, xpath)
     end
   end
 
+  def get_structured_date(doc, xpath)
```

Comment:
There appear to be minor differences between how the UI structured date parser assigns values that end up in the fixture file, versus how the converter tool assigns some values. 

We don't need to be testing the function of the structured date parser in our individual module tests, so we just check that the structured date field structure was created under the expected xpath, and that the dateDisplayDate is what was in the CSV column.